src/pages/Profile/Profile.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { FC } from 'react'
+import { Typography, Grid, Paper, TextField } from '@material-ui/core'
+
+import BackgroundImage from 'assets/images/background.jpg'
+import { useStyles } from './styles'
+
+type Props = {
+  userName: string
+  userLastname: string
+  email: string
+  teamName: string
+  developerRole: string
+}
+
+const Profile: FC<Props> = ({ userName, userLastname, email, teamName, developerRole }: Props) => {
+  const classes = useStyles()
+
+  function renderTitle() {
+    const titleText = 'My Profile'
+    return (
+      <Typography component="h1" variant="h5">
+        {titleText}
+      </Typography>
+    )
+  }
```

Comment:
Also for readability reasons you should always add empty line between different logic code:
```suggestion
  function renderTitle() {
    const titleText = 'My Profile'
    
    return (
      <Typography component="h1" variant="h5">
        {titleText}
      </Typography>
    )
  }
```